examples/pwm/sources/zynq_video_board.xdc

```diff
@@ -0,0 +1,3 @@
+set_property IOSTANDARD LVCMOS33 [get_ports pwm]
+#set_property PACKAGE_PIN N22 [get_ports pwm]
```

Comment:
Should this be removed?

Reply:
Yes, the unneeded line was deleted and the filename is `zynq.xdc` now